internal/config/config_test.go

```diff
@@ -152,8 +150,8 @@ func standardProduct() crt.Product {
 			Core: "1.2.3",
 			Meta: "",
 		},
-		SourceHash:   "deadbeef",
-		Revision:     "cabba9e",
+		SourceHash:   "<source-hash>",
+		Revision:     "<commit-sha>",
```

Comment:
This change is repeated elsewhere. It makes test data more meaningful without affecting the correctness of the tests.